src/app/my-account/my-account/my-account.component.html

```diff
@@ -44,8 +44,8 @@ <h1 class="header-2 font-darkblue fw-bold">{{ "MY_ACCOUNT" | translate }}</h1>
             icon="translate"
           ></app-search-result-attribute>
 
-          <div class="pt-2 pt-sm-0 ps-sm-2" buttons>
-            <button type="button" class="btn btn-link btn-sm" (click)="removeAnnouncement(announcement)">
+          <div class="pt-2 pt-sm-0 ps-sm-2" buttons (click)="removeAnnouncement(announcement, $event)">
+            <button type="button" class="btn btn-light btn-sm">
```

Comment:
Button without a `click` handler and div with one. I know we're cutting corners but this seems like too much... CSS can be adjusted if that's the issue.